### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kq333**
[TEST REPORT LINK](https://kq333.github.io/layout_html-form/report/html_report/)
[DEMO LINK](https://kq333.github.io/layout_html-form/)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 module.exports = {
```
Comment (michalcywka) [inline .eslintrc.js]: Don't change this file please :)

File: .linthtmlrc.json [hunk 2]
```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```
Comment (michalcywka) [inline .linthtmlrc.json]: Don't change this file please :)

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,188 @@
...
+        </div>
+
+        <div>
+          <label for="time">What time do you go to bed?</label>
+          <input
+              type="date"
+              id="time"
+              name="time"
+              required
+              placeholder="dd / mm / yyyy"
```
Comment (michalcywka) [inline src/index.html]: Too much indentation

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,188 @@
...
+        <legend>Additional info:</legend>
+
+        <label for="comments">Comments:
+        </label>
+        <textarea
+          id="comments"
+          name="comments"
+          rows="2"
+          column="10"
+        >
```
Comment (michalcywka) [inline src/index.html]: Please have a look at notes in readme.md

File: src/style.css [hunk 5]
```diff
@@ -1 +1,13 @@
 /* styles go here */
+
+fieldset {
```
Comment (michalcywka) [inline src/style.css]: Checklist point 6

**Review by michalcywka — CHANGES_REQUESTED** [review]
I marked some of the mistakes,  please have a careful look at the checklist. Also please do not commit ```package-lock.json``` file.

File: .linthtmlrc.json (hunk 2 above)
Comment (kq333) [reply .linthtmlrc.json]: I have not changed this file.

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="http://localhost:3000/api" method="post">
```
Comment (michalcywka) [inline src/index.html]: Action should be something else (readme.md)

**Review by michalcywka — CHANGES_REQUESTED** [review]
Two links with GitHub Pages that are required in the description are missing.

**Review by michalcywka — CHANGES_REQUESTED** [review]
Please read carefully point 19 from [tutorial](https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github) Links in the description are missing.

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,186 @@
...
+        <legend>Additional info:</legend>
+
+        <label for="comments">Comments:
+        </label>
+        <textarea
+          id="comments"
+          name="comments"
+          rows="2"
+          column="10"
+        ></textarea>
```
Comment (michalcywka) [inline src/index.html]: Please have a closer look at the readme.md file on how to fix this issue

File: src/style.css [hunk 8]
```diff
@@ -1 +1,13 @@
 /* styles go here */
+
+.fieldset {
+  margin-bottom: 20px;
+}
+
+.fieldset > div:not(:last-of-type) {
```
Comment (michalcywka) [inline src/style.css]: Checklist point 6
It would be better to use another class (similar to 'comments-rec') and use it in divs around inputs and labels.

**Review by michalcywka — CHANGES_REQUESTED** [review]
Please look at Pull Requests from other people and see how their description looks like. The links from description here in the PR are still missing.

**Review by michalcywka — CHANGES_REQUESTED** [review]
Please see my message on Slack.

**Comment by kq333** [comment]
[DEMO LINK](https://kq333.github.io/layout_html-form/)
[TEST REPORT LINK](https://kq333.github.io/layout_html-form/report/html_report/)

File: src/index.html (hunk 7 above)
Comment (michalcywka) [reply src/index.html]: This needs to be fixed :) Look at Tips&Hints in readme.md, point 4

File: src/index.html [hunk 9]
```diff
@@ -11,7 +11,184 @@
...
+            id="sasquatch"
+            name="monster"
+            value="#000000"
+          >
+        </div>
+
+        <div class="labels">
+          <label for="time">What time do you go to bed?</label>
+          <input
+              type="date"
```
Comment (michalcywka) [inline src/index.html:135]: Checklist point 3

File: .linthtmlrc.json (hunk 2 above)
Comment (michalcywka) [reply .linthtmlrc.json]: I understand, this is a side effect of `npm install`. Nevertheless you should only commit files that you have changed yourself. One way to fix this is to copy the original content of the file and commit it again :)

**Review by michalcywka — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 10]
```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
```
Comment (michalcywka) [inline src/index.html]: Checklist point 1 (make it look like checklist point 3)

File: src/index.html [hunk 11]
```diff
@@ -11,7 +11,182 @@
...
+            title="color"
+          >
+        </div>
+
+        <div class="labels">
+          <label for="time">What time do you go to bed?</label>
+          <input
+              type="date"
+              id="time"
+              step="2"
```
Comment (michalcywka) [inline src/index.html]: Checklist point 3

File: src/index.html [hunk 12]
```diff
@@ -11,7 +11,182 @@
...
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="comments">Comments:
+
+          <textarea
+          name="coments"
```
Comment (michalcywka) [inline src/index.html]: checklist point 3

**Review by michalcywka — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 13]
```diff
@@ -11,7 +11,183 @@
...
+            <option value="yes" selected>yes</option>
+            <option value="no">no</option>
+          </select>
+        </div>
+      </fieldset>
+    </form>
+
+    <br>
+    <br>
+    <button type="submit">Submit</button>
```
Comment (michalcywka) [inline src/index.html]: Button should be in form, don't use <br> tags to make space :)

File: src/index.html [hunk 14]
```diff
@@ -11,7 +11,183 @@
...
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="comments">Comments:
+
+          <textarea
```
Comment (michalcywka) [inline src/index.html:173]: Please fix linter error here.

File: src/index.html [hunk 15]
```diff
@@ -11,7 +11,183 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
```
Comment (michalcywka) [inline src/index.html]: If you do it like this, attributes need to follow checklist point 3 also

**Review by michalcywka — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 16]
```diff
@@ -11,7 +11,183 @@
...
+        </div>
+
+        <div class="labels">
+          <label for="cars">What are your favorite brand of cars?</label>
+
+          <select
+            name="cars"
+            id="cars"
+            required
+            multiple>
```
Comment (michalcywka) [inline src/index.html]: Checklist point 3

File: src/index.html [hunk 17]
```diff
@@ -11,7 +11,183 @@
...
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="comments">Comments:
+          <textarea
+            name="coments"
+            autocomplete="off"
+          ></textarea>
```
Comment (michalcywka) [inline src/index.html]: There is still a problem here.

**Review by michalcywka — CHANGES_REQUESTED** [review]

**Review by michalcywka — CHANGES_REQUESTED** [review]
Still needs textarea fix

**Review by michalcywka — CHANGES_REQUESTED** [review]
Still not all the necessary files are reverted

**Review by michalcywka — CHANGES_REQUESTED** [review]
Unnecessary files are still present in the Pull Request.

**Review by michalcywka — APPROVED** [review]

**Pull request closed without merging**